Makefile

```diff
@@ -317,11 +317,11 @@ ifdef JUNIT
 endif
 
 .PHONY: test-check
-test-check: test-lint test-boilerplate test-helm verify-vendor-licenses ## Run all static checks.
+test-check: test-boilerplate test-helm verify-vendor-licenses ## Run all static checks.
 
-.PHONY: test-lint
-test-lint: ## Run golint test.
-	hack/verify-golint.sh
+.PHONY: lint
```

Comment:
Nit: I prefer the old name `test-lint`. No strong opinion.

Reply:
To me, `test` doesn't link to `lint` and I want to keep it short. 😄